README.md

```diff
@@ -12,3 +12,20 @@ page](https://docs.github.com/en/get-started/writing-on-github/working-with-adva
 might help with the notation for mathematical expressions.
 
 $T(n) \in O(f(n)) \iff \exists c, n_0: T(n) \leq c \cdot f(n) \forall n \geq n_0$
+
+Thm : $T(\log_{2} n) \in O(\log_{5} n) \iff \exists c, n_0: T(\log_{2} n) \leq c \cdot (\log_{5} n) \forall n \geq n_0$
```

Comment:
It looks like you're assuming a specific $T(n)$ here. You can't do that for a general proof.